magma/array.py

```diff
@@ -388,14 +398,17 @@ def iswhole(self, ts):
 
         return True
 
+    def iswhole(self):
+        return Array._iswhole(self.ts)
```

Comment:
Why this pattern? (Internally using `SpecificClass._static_method`)
If you are going to do this its seems better to minimally use `type(self)._static_method`

Reply:
This seems like a reasonable change to preserve backwards compatibility (the old `iswhole` method is still available, but from now on we should be using the _iswhole static method).

Reply:
Ah, scratch that, I see that the is whole signature has changed to not accept any parameters,  I think this is okay, the static method could also be a private helper but a static method is reasonable to me.

Reply:
Yeah, it was kind of a broken interface anyway and not really used outside, and self-contained.